### PR TITLE
pylintrc file from Google style guide

### PR DESCRIPTION
**Pull request by rpgolshan**
Same as stock Google style guide, except replace two spaces with four

### CODE REVIEW

File: frontend/pylintrc [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+
+[MASTER]
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=third_party
+
+# Add files or directories matching the regex patterns to the blacklist. The
+# regex matches against base names, not paths.
+ignore-patterns=
```
Comment (xyzsam) [inline frontend/pylintrc:16]: add `.*pb2.py` to exclude generated proto files.

**Review by xyzsam — APPROVED** [review]

**Comment by xyzsam** [comment]
One thing to note - access to protected members is common in unit tests, but pylint doesn't have a "test" mode to conditionally disable this check, so I've disabled this warning manually in each of my unit tests.

**Comment by xyzsam** [comment]
One other thing - your naming conventions for class member functions (lower_case) is different from the Google style guide (CamelCase) - so this linter will throw a lot of errors. I'm not sure if that's something you want to address? AFAIK, yapf does not rename functions for you.

File: frontend/pylintrc (hunk 1 above)
Comment (rpgolshan) [reply frontend/pylintrc]: Done.

**Review by xyzsam — APPROVED** [review]

**Pull request merged**